modules/ocf_mirrors/manifests/timer.pp

```diff
@@ -12,9 +12,19 @@
   # Any meaningful second precision requires AccuracySec to be set
   $on_calendar = "${year}-${month}-${day} ${hour}:${minute}:00"
 
+  if !($type in ['sync', 'monitor']) {
+    fail('Type parameter of ocf_mirrors:timer must be either sync or monitor')
+  }
+
+  if $type == 'sync' {
```

Comment:
Not a huge deal, but if statements in Puppet are a code smell. The Puppet resource for sync jobs and monitoring jobs should be separate since they do totally different things.